src/components/GridTable.test.tsx

```diff
@@ -55,11 +55,6 @@ const nestedColumns: GridColumn<NestedRow>[] = [
 ];
 
 describe("GridTable", () => {
-  it("renders", async () => {
```

Comment:
@stephenh are you ok with deleting this snapshot test since we have stories?

Reply:
Yep!